examples/siro_sandbox/sandbox_app.py

```diff
@@ -538,14 +539,12 @@ def _save_recorded_keyframes_to_file(self):
     def _update_cursor_style(self, post_sim_update_dict):
         do_update_cursor = False
         if self._cursor_style is None:
-            if self._first_person_mode:
-                self._cursor_style = Application.Cursor.HIDDEN_LOCKED
-            else:
-                self._cursor_style = Application.Cursor.ARROW
+            self._cursor_style = Application.Cursor.ARROW
```

Comment:
Defaulting to arrow cursor for first-person mode. This change is essentially Ruta's recent request: that we disable mouse-look.